admin/database/database.go

```diff
@@ -151,6 +152,17 @@ type Project struct {
 	ProductionDeploymentID *string   `db:"production_deployment_id"`
 	CreatedOn              time.Time `db:"created_on"`
 	UpdatedOn              time.Time `db:"updated_on"`
+	Envs                   Envs      `db:"env"`
```

Comment:
1. This should be either `Env` (singular) or `Variables` (plural). I have been wondering if we should rename it to "variables" everywhere, since people seem to get confused between this and real environment variables.
2. Let's stick with the terminology above and make it `ProductEnv` (or `ProductionVariables`). Makes it easier to add preview support later on
3. nit: Move the var up in the type after `ProductionBranch` (for natural grouping of related fields)